### PR TITLE
docs: improved contents

### PR DESCRIPTION
**Pull request by ajile-in**
- Improved sidebar menu
- updated starlight package to latest
- improved docs contents

### CODE REVIEW

File: src/content/docs/en/guides/local-setup-using-docker.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 ---
-title: Local Setup using NPM
+title: Install CREDEBL using Docker
 description: A reference page in my new CREDEBL docs site.
```
Comment (vivekayanworks) [inline src/content/docs/en/guides/local-setup-using-docker.md:3]: We need to modify this content.

File: src/content/docs/en/guides/local-setup-using-npm.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
 ---
-title: Local Setup
+title: Install CREDEBL from repository
 description: A reference page in my new CREDEBL docs site.
```
Comment (vivekayanworks) [inline src/content/docs/en/guides/local-setup-using-npm.md:3]: Please modify the content.

**Review by vivekayanworks — CHANGES_REQUESTED** [review]

File: src/content/docs/en/guides/local-setup-using-docker.md (hunk 1 above)
Comment (ajile-in) [reply src/content/docs/en/guides/local-setup-using-docker.md:3]: @vivekayanworks - Can you suggest please the specific changes?

Comment (vivekayanworks) [reply src/content/docs/en/guides/local-setup-using-docker.md:3]: @ajile-in In the description we need to remove "my new".

File: src/content/docs/en/guides/local-setup-using-npm.md (hunk 2 above)
Comment (vivekayanworks) [reply src/content/docs/en/guides/local-setup-using-npm.md:3]: @ajile-in In the description we need to remove "my new".

**Pull request merged**